Python/Product/TestAdapter/visualstudio_py_testlauncher.py

```diff
@@ -236,19 +236,15 @@ def main():
     parser = OptionParser(prog = 'visualstudio_py_testlauncher', usage = 'Usage: %prog [<option>] <test names>... ')
     parser.add_option('-s', '--secret', metavar='<secret>', help='restrict server to only allow clients that specify <secret> when connecting (legacy debugger only)')
     parser.add_option('-p', '--port', type='int', metavar='<port>', help='listen for debugger connections on <port>')
-    parser.add_option('-d', '--debugger-search-path', type='str', metavar='<debugger_path>', help='Path to the debugger directory')
     parser.add_option('-x', '--mixed-mode', action='store_true', help='wait for mixed-mode debugger to attach')
     parser.add_option('-t', '--test', type='str', dest='tests', action='append', help='specifies a test to run')
     parser.add_option('-c', '--coverage', type='str', help='enable code coverage and specify filename')
     parser.add_option('-r', '--result-port', type='int', help='connect to port on localhost and send test results')
     parser.add_option('--test-list', metavar='<file>', type='str', help='read tests from this file')
     parser.add_option('--dry-run', action='store_true', help='prints a list of tests without executing them')
     (opts, _) = parser.parse_args()
-    
-    sys.path[0] = os.getcwd()
-    if opts.debugger_search_path:
-        sys.path.insert(0, opts.debugger_search_path)
 
+    sys.path.insert(1, os.getcwd())
```

Comment:
oh is it because at [0] it's the path to this script and if you overwrite that then you can't load debugpy relative to this script?

Reply:
The value "sys.path[0]" is the directory of the python script that was started. The old logic was concluded because it was changing the starting script directory, then adding the debugger path, if required (in the following "if opts.debugger..." statement). The "os.getcwd()" refers to the directory of the user's python script which has simply been inserted in.